content/applications/inventory_and_mrp/purchase/advanced/edi.rst

```diff
@@ -0,0 +1,112 @@
+==================================
+EDI purchase-to-sales order import
+==================================
+
+.. |EDI| replace:: :abbr:`EDI (Electronic Data Exchange)`
+.. |PO| replace:: :abbr:`PO (purchase order)`
+.. |SO| replace:: :abbr:`SO (sales order)`
+
+Electronic data interchange (EDI) enables companies using different software systems to exchange
+information in a standardized, structured format.
+
+In Odoo, a *purchase order* (PO) can be exported as an XML file and imported as a *sales order* (SO)
+into another Odoo database, removing the need for manual entry of products, quantities, prices, and
+other key information.
+
+.. note::
+   Exported XMLs follow the `Order-X schema <https://fnfe-mpe.org/factur-x/order-x/>`_, a standard
+   commonly used in European countries. When exchanging data between two Odoo databases, this schema
+   remains compatible.
+
+   However, implementing custom developments for software that does not support the Order-X schema
+   may introduce additional complexity.
+
+Roles and configuration
+=======================
+
+To facilitate the |EDI| workflow, two companies are involved: the buyer (the company placing the
+order) and the seller (the company fulfilling the order). Each company has specific roles and
+configurations:
```

Comment:
Formatting: the colon to me indicates that a list of items is going to follow (either inline in the paragraph or in a list of some kind), so I'd end this on a period if we want to keep the h3 heading formats as they are, or turn those paragraphs below into list items.

```suggestion
configurations.
```